datasets/GM0307_amersfoort/GM0307_amersfoort.derived.ad

```diff
@@ -126,37 +104,59 @@
 - heat_demand_reduction_high_insulation_terraced_house = 69.0
 - heat_demand_reduction_high_insulation_apartments = 67.0
 - heat_demand_reduction_high_insulation_buildings = 74.0
+- heat_share_of_apartments_with_block_heating = 0.15
+- heat_households_indoor_pipelines_investment_costs_with_block_heating_eur = 2022.0
+- heat_households_indoor_pipelines_investment_costs_without_block_heating_eur = 5984.0
+- heat_buildings_indoor_investment_costs_eur_per_kw = 151.0
+- heat_buildings_heat_meter_investment_costs_eur_per_kw = 1.35
+- heat_buildings_heat_meter_investment_costs_eur_per_connection = 913.0
+- heat_exchanger_station_investment_costs_eur_per_kw = 80.0
+- heat_sub_station_investment_costs_eur_per_kw = 135.0
+- heat_distribution_pipelines_investment_costs_eur_per_meter = 1200.0
+- heat_length_of_distribution_pipelines_in_meter_per_residence_object_first_bracket = 3.9
+- heat_length_of_distribution_pipelines_in_meter_per_residence_object_second_bracket = 4.6
+- heat_length_of_distribution_pipelines_in_meter_per_residence_object_third_bracket = 5.2
+- heat_length_of_distribution_pipelines_in_meter_per_residence_object_fourth_bracket = 5.8
+- heat_length_of_distribution_pipelines_in_meter_per_residence_object_fifth_bracket = 7.3
+- heat_length_of_connection_pipelines_in_meter_per_residence_first_bracket = 1.6
+- heat_length_of_connection_pipelines_in_meter_per_residence_second_bracket = 3.0
+- heat_length_of_connection_pipelines_in_meter_per_residence_third_bracket = 4.2
+- heat_length_of_connection_pipelines_in_meter_per_residence_fourth_bracket = 5.2
+- heat_length_of_connection_pipelines_in_meter_per_residence_fifth_bracket = 6.5
+- heat_primary_pipelines_investment_costs_per_kw = 200.0
+- heat_yearly_indoor_infrastructure_maintenance_costs_factor = 0.025
+- heat_yearly_outdoor_infrastructure_maintenance_costs_factor = 0.03
 - electric_vehicle_profile_1_share = 1.0
 - electric_vehicle_profile_2_share = 0.0
 - electric_vehicle_profile_3_share = 0.0
 - electric_vehicle_profile_4_share = 0.0
 - electric_vehicle_profile_5_share = 0.0
 - solar_pv_profile_1_share = 1.0
 - solar_pv_profile_2_share = 0.0
-- annual_infrastructure_cost_gas = 12.49342089656725
+- annual_infrastructure_cost_gas = 12.730893293642405
 - areable_land = 19.75
-- co2_emission_1990 = 0.793494190665536
+- co2_emission_1990 = 0.788439123863016
 - coast_line = 0.0
 - interconnector_capacity = 60000.0
 - total_land_area = 62.77
-- number_of_buildings = 9623.0
-- number_of_cars = 98852.0
-- number_of_residences = 65289.0
-- number_of_inhabitants = 153602.0
-- offshore_suitable_for_wind = 448.608030215936
+- number_of_buildings = 4932.0
+- number_of_cars = 102990.0
+- number_of_residences = 66530.0
+- number_of_inhabitants = 155226.0
+- offshore_suitable_for_wind = 12500.0
```

Comment:
The offshore area suitable for wind seems to have increased strongly. Is this as expected?

Reply:
It is now the same value for all municipalities: 12500 square kms.

Reply:
> It is now the same value for all municipalities: 12500 square kms.

Interesting.. This seems to be the case for all 2018 datasets in the dataset manager. @michieldenhaan would you have any idea what's going on here?

Reply:
If I remember correctly we first scaled the potential based on number of residences but after receiving complaints that this was too strict in some scenarios we decided to allocate the full NL potential to each municipality (to ensure the slider max is always sufficient)

Reply:
That explains, thanks!